.dockerignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+build
+.dockerignore
+Dockerfile
```

Comment:
Another good candidate to ignore is the `.git/` folder, as well as the `.gitignore` file.